### DISCUSSION TITLE
Donation possible?

### DISCUSSION THREAD

**Discussion by kaladinsb0** [Taiizor/ReaLTaiizor#92] [Q&A] [answered]
I was wondering if there was anywhere that donations can be made for this project. I don't use GitHub much so might have missed it if there is a way directly on this page.

**Comment by Taiizor** [comment, marked as answer]
> I was wondering if there was anywhere that donations can be made for this project. I don't use GitHub much so might have missed it if there is a way directly on this page.

There are currently no places to donate for this project. But still, thank you for your opinion.
